packages/mobile/src/utils/formatting.ts

```diff
@@ -32,16 +32,47 @@ export const getFeeDisplayValue = (value: BigNumber.Value | null | undefined): s
   return value ? numeral(BigNumber.max(value, 0.001).toNumber()).format('0[.][0000]') : ''
 }
 
+/**
+ * More precise getFeeDisplayValue with built in rounding
+ * Used for small Network Fees in transaction feed
+ * @param value fee amount
+ */
+export const getNetworkFeeDisplayValue = (value: BigNumber.Value): string => {
+  const roundedNumber = new BigNumber(value)
+  if (roundedNumber.isLessThan(0.001)) {
+    return '<0.001'
+  } else {
+    return numeral(roundUp(value, 3).toNumber()).format('0[.][000]')
+  }
+}
+
+/**
+ * Even more precise getFeeDisplay value for Network Fee drilldown
+ * Built in rounding below <0.000001. Displays fee to 6 decimal
+ * places if less than 0.001, else to 3 decimal places
+ * @param value fee amount
+ */
+export const getPreciseNetworkFeeDisplayValue = (value: BigNumber.Value): string => {
+  const roundedNumber = new BigNumber(value)
+  if (roundedNumber.isLessThan(0.000001)) {
+    return '<0.000001'
+  } else if (roundedNumber.isLessThan(0.001)) {
+    return numeral(roundUp(value, 6).toNumber()).format('0[.][000000]')
+  } else {
+    return numeral(roundUp(value, 3).toNumber()).format('0[.][000]')
+  }
+}
+
 export const divideByWei = (value: BigNumber.Value, decimals?: number) => {
   const bn = new BigNumber(value).div(WEI_PER_CELO)
   return decimals ? bn.decimalPlaces(decimals) : bn
 }
 
-export function roundDown(value: BigNumber.Value, decimals: number = 2) {
+export function roundDown(value: BigNumber.Value, decimals: number = 2): BigNumber {
```

Comment:
Why was this explicit typing necessary?

Reply:
Removed, had added it as I found the switching between BigNumber.value's and BigNumber's confusing, but can just hover over for the type signature

Reply:
Actually, going to keep this in to be consistent with the rest of the files explicit typing along with the potential BigNumber.value vs BigNumber confusion